components/core/src/clp/ffi/ir_stream/Serializer.cpp

```diff
@@ -493,37 +489,43 @@ auto Serializer<encoded_variable_t>::serialize_val(
 // file
 template auto Serializer<eight_byte_encoded_variable_t>::create(
 ) -> OUTCOME_V2_NAMESPACE::std_result<Serializer<eight_byte_encoded_variable_t>>;
+
```

Comment:
Why all the newlines here?

Reply:
Sorry, I thought it was added by clang-format. I will remove them